homeassistant/components/vicare/water_heater.py

```diff
@@ -54,7 +56,7 @@
 }
 
 
-def _get_circuits(vicare_api):
+def _get_circuits(vicare_api: PyViCareDevice) -> list[Any]:
```

Comment:
Isn't there a class we can reference here?

Reply:
This should be `HeatingCircuit` if I [read the code](https://github.com/somm15/PyViCare/blob/8ba411483a865e074d1146fd1b8b7a8c4f4be122/PyViCare/PyViCareHeatingDevice.py#L47C9-L51) correctly.

```suggestion
def _get_circuits(vicare_api: PyViCareDevice) -> list[HeatingCircuit]:
```